.github/workflows/android-ci.yml

```
@@ -0,0 +1,74 @@
+# CI 목적
+# - Kotlin 코드 품질 검증 (detekt)
+# - Android / Compose Lint 검사
+# - 초기엔 빌드/테스트 제외
+
+name: CI
+
+# workflow 실행 제어
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)